examples/gallery/textures/app.js

```diff
@@ -0,0 +1,161 @@
+/* global document */
```

Comment:
The `gallery` folder is intended for demos that are self contained scripts that do not need transpilation. 

Since this example has been split into multiple files (as opposed to one index.html with `<script>` section) you can't open it without webpack or running an http-server, so we should think about moving this into another folder.

Another strategy is to add

Reply:
Moved all files into separate `examples/textures-tester` folder.